lib/protokoll/counter.rb

```diff
@@ -30,7 +29,7 @@ def self.build_attrs(object, options)
     end
 
     def self.outdated?(record, options)
-      Time.now.utc.strftime(update_event(options)).to_i > record.updated_at.strftime(update_event(options)).to_i
+      Time.zone.now.strftime(update_event(options)).to_i > record.updated_at.strftime(update_event(options)).to_i
```

Comment:
```suggestion
      Time.current.strftime(update_event(options)).to_i > record.updated_at.strftime(update_event(options)).to_i
```

https://apidock.com/rails/Time/current/class